packages/block-library/src/navigation/test/use-navigation-menu.js

```diff
@@ -8,6 +8,7 @@ import { store as coreStore } from '@wordpress/core-data';
  * Internal dependencies
  */
 import useNavigationMenu from '../use-navigation-menu';
+import { SELECT_NAVIGATION_MENUS_ARGS } from '../constants';
```

Comment:
I know this might seem like a massive nit, but I would really prefer to avoid relying on app code to provide abstraction in tests.

Whilst it seems good now, further down the line it could lead to unexpected consequences with tests "automatically" adapting their behaviour when the production code changes. 

I'd much prefer to see these arguments inlined and have a tonne of duplication which keeps things explicit and avoids abstraction in tests 🙏 

```suggestion
```